tonos_ts4/abi.py

```diff
@@ -133,6 +133,10 @@ def check_method_params(abi, method, params):
     for param in inputs:
         pname = param['name']
         if pname not in params:
+            # the default for 'responsible' functions
+            if pname == 'answerId':
+                res[pname] = 0
```

Comment:
Hi. This does not seem to be correct fix. What did you mean?
Please feel free to join us at Telegram @ton_ts4 and ask more questions, if needed.

Reply:
I assume `answerId` an infrastructure parameter, not an application parameter. Passing it when you test a responsible function via external call is rather a boilerplate code and so is a subject to be tackled with by the test framework.

The intention is to get rid of this when you don't need need this explicitly. 
```
result = c.call_method('method', dict(answerId = 0))
result = c.call_method('method')
```
Both lines produce the same result, but the latter looks cleaner. The `answerId` here tells us nothing about the logic we wnat to test. Now omitting the `answerId` for responsible function gives an exception.

Doesn't it seem like an argument?